Nodejs/Setup/Custom_Action/CustomAction.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Deployment.WindowsInstaller;
+using System.Diagnostics;
+using System.Net;
+using System.Collections.Specialized;
+using System.Web.Script.Serialization;
+
+namespace Custom_Action
+{
+    public class CustomActions
+    {
+        [CustomAction]
+        public static ActionResult RecordInstallStartTime(Session session)
+        {
+            session["InstallStartTime"] = DateTime.Now.ToString();

```

Comment:
This represents the current date/time on the local computer, and ToString() is culture specific.